app/models/comment.rb

```diff
@@ -57,7 +57,6 @@ class Comment < ApplicationRecord
   validates :positive_reactions_count, presence: true
   validates :public_reactions_count, presence: true
   validates :reactions_count, presence: true
-  validates :user_id, presence: true
```

Comment:
I think this was removed by Rubocop ([`Rails/RedundantPresenceValidationOnBelongsTo`](https://docs.rubocop.org/rubocop-rails/cops_rails.html#railsredundantpresencevalidationonbelongsto)), I didn't touch it.

Reply:
I've seen that one trigger as well when I modified a model.